src/analysis/symbols.rs

```diff
@@ -18,7 +18,7 @@ impl Symbol {
     pub fn parent(&self) -> String {
         let mut ret = String::new();
         if Some("gobject") == self.crate_name() {
-            ret.push_str("glib::object::");
+            ret.push_str("glib::");
```

Comment:
Did it move or do we have a better object path now?

Reply:
it's used for a ton of cases like `glib::object::Value` which is a wrong link because Value is in `glib::value::Value` or `glib::object::ObjectExt` which is already in the prelude. So I removed it :)